specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2021-03-01-preview/appplatform.json

```diff
@@ -384,6 +497,126 @@
         }
       },
       "x-ms-external": true
+    },
+    "BuildResource": {
```

Comment:
Shall we expose `BuildResource` or `BuildDefinition` to external user? Use `Resource` as suffix is not the paradigm in our swagger file.